docs/source/usage/config.rst

```diff
@@ -349,7 +350,7 @@ Training settings
 -----------------
 
 -  ``optimizer``: Specify which optimizer to use. Currently supported
-   is Adam (standard). New optimizers can be added
+   are Adam (default) and AdamW. New optimizers can be added
```

Comment:
The 
```suggestion
   are Adam and AdamW. New optimizers can be added
```
If I understand our `get_optimizer` function correctly, then Adam is not a default optimizer. The optimizer has to be set explicitly by the user, otherwise https://github.com/neuralhydrology/neuralhydrology/blob/master/neuralhydrology/utils/config.py#L620 will throw an error.